data/sql_updates/create_totals_presidential_view.sql

```diff
@@ -1,3 +1,4 @@
+drop view if exists ofec_totals_presidential_vw;
```

Comment:
This hasn't been a view before so we don't need this line. It is the same case for create_totals_pacs_parties_view.sql and create_totals_house_senate_view.sql